facebook/__init__.py

```diff
@@ -255,6 +255,12 @@ def request(
                 args["access_token"] = self.access_token
 
         try:
+            import logging
+            logging.info('GRAPH REQUEST URL:%s'% (FACEBOOK_GRAPH_URL + path))
+            logging.info('GRAPH REQUEST params:')
+            logging.info(args)
+            logging.info('GRAPH REQUEST data:')
+            logging.info(post_args)
```

Comment:
Please read the [contributing guidelines](https://github.com/mobolic/facebook-sdk/blob/master/CONTRIBUTING.rst).

> A pull request should only include commits directly applicable to its change.

Reply:
thanks @martey